src/schema/v2/partner/partner.ts

```diff
@@ -115,6 +118,11 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
     description:
       "Only allowed for authorized admin/partner requests. When false fetch :all properties on an artwork, when true or not present fetch artwork :short properties",
   },
+  visibilityLevels: {
+    type: GraphQLList(ArtworkVisibility),
```

Comment:
I _think_ this should be `new GraphQLList(ArtworkVisibility)`

Reply:
oh yes yes you are right, one sec 🙏🏻